### DISCUSSION TITLE
Build error

### DISCUSSION THREAD

**Discussion by jesusglezalv** [Builty/TonexOneController#40] [Q&A] [answered]
Hi,

first thanks to @Builty for such a great  project. I'm trying to set up the development environment to understand how the solution works and help if possible.

Long time ago since I worked with HW, but I think I have the right background. I've setup all the  ESP-IDF development environment and I've try some of the examples on a Waveshare 4.3" B LCD board  and everything seems to work. Also I tried some of the TonexOneController releases on the board and I was able to download the FW and make the board work.

Therefore it seems both the dev environment and the HW works. Then I fetched from GitHub the latest release (1.0.3.2) and when I try to build the project I got the following error:

> `CMake Error: The source directory "/Users/xxxxx/src/Tonex/TonexOneController" does not appear to contain CMakeLists.txt.
> Specify --help for usage, or press the help button on the CMake GUI.`

Any hints what step I missed up??

Thanks in advance!!

**Comment by Builty** [comment, marked as answer]
Hi Jesus. 
The CmakeLists.txt file is actually in the source folder. Which folder did you select when you opened it in VS Code? It needs to be the folder "... TonexOneController\source"

I think that might be issue you have.

**Reply by jesusglezalv** [reply]
Thanks Builty. That was definitely my mistake. Now I'm able to build the project.
